sources/api/storewolf/src/main.rs

```diff
@@ -385,7 +393,8 @@ fn populate_default_datastore<P: AsRef<Path>>(
     // If any other defaults remain (configuration files, services, etc),
     // write them to the datastore in Live state
     debug!("Serializing other defaults and writing new ones to datastore");
-    let defaults = to_pairs(&defaults_val).context(error::SerializationSnafu {
+    let defaults_json = serde_json::to_value(defaults_val).context(error::JsonConversionSnafu)?;
+    let defaults = to_pairs(&defaults_json).context(error::SerializationSnafu {
```

Comment:
The `configuration-files` and `services` I mentioned should be handled here.

I think something that would be extremely helpful would be some integration-level checks of storewolf that the resulting datastore from a given set of defaults is as-expected. This would provide a ton of validation to the datastore serializer.

We could work on that in a followup though.